ksqldb-engine/src/main/java/io/confluent/ksql/services/DefaultConnectClient.java

```diff
@@ -285,10 +291,25 @@ public DefaultConnectClient(
     return authHeader;
   }
 
+  @VisibleForTesting
+  Map<String, String> getAdditionalRequestHeaders() {
+    return additionalRequestHeaders;
+  }
+
   private Header[] headers() {
```

Comment:
nit: it appears to me that both `authHeader` and `additionalRequestHeaders` do not change over time. Is this right? If so, we can probably create `Header[]` once and just serve it from this method.

Reply:
Yeah, this is true. The implementation was already this way (separate `headers()` method on each request) before my changes so I didn't touch it. I can look into refactoring, either as part of this PR or as a follow-up.

Reply:
Made this change as part of this PR.

Reply:
Thank you!